packages/react/src/components/header/components/headerUniversalBar/HeaderUniversalBar.module.scss

```diff
@@ -2,6 +2,7 @@
 
 .headerUniversalBar {
   align-items: center;
+  background: var(--universal-bar-background-color);
```

Comment:
This could be in .headerUniversalBarContainer as it has background style already.